grafana/dashboards/deployment.json

```diff
@@ -660,245 +660,253 @@
       }
     },
     {
-      "collapsed": true,
+      "collapsed": false,
```

Comment:
Most of the changes in this file are a result of exporting the dashboard when a row that was collapsed by default is expanded. That causes panels that would normally be nested to instead appear as separate rows in the JSON. I think the easiest fix is to load this branch in grafana, collapse all of the rows that changed from `"collapsed": true` to `"collapsed": false`, and re-export.